src/tests/lxc-test-unpriv

```diff
@@ -27,6 +27,42 @@ if [ $(id -u) -ne 0 ]; then
 	echo "ERROR: Must run as root."
 	exit 1
 fi
+
+# Test if we're using an overlayfs module that handles symlinks correctly. If
+# not, we skip these tests since overlay clones will not work correctly.
+OVERLAYFS=$(modprobe -q overlayfs)
```

Comment:
You don't actually use ${OVERLAYFS} anywhere below, so you probably should just make this:

    if modprobe -q overlayfs; then